test/built-ins/Promise/anySettled/S25.4.4.3_A1.1_T1.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2014 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: Promise.anySettled is callable
+es6id: S25.4.4.3_A1.1_T1
+author: Sam Mikes
+description: Promise.anySettled is callable
+---*/
```

Comment:
some meta here: for all the files in this `Promise.anySettled` folder please replace any es6id with `esid: sec-promise-anysettled` and make sure each one of them has the `features: [Promise.anySettled]`, adding the feature tag if they already have a `features` entry.